src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
```

Comment:
It's look like correct version, but I think you need to use destructuring

Reply:
[Checklist](https://github.com/mate-academy/react_person/blob/master/checklist.md) p.1